docs/diagrams/FilterSequenceDiagramPart2.puml

```diff
@@ -0,0 +1,42 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant "f:FilterCommand" as FilterCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "p:ElderlyHasTagPredicate" as ElderlyHasTagPredicate MODEL_COLOR
+end box
+
+[-> LogicManager
+activate LogicManager
+
+LogicManager -> FilterCommand : execute()
+activate FilterCommand
+
+FilterCommand -> Model : updateFilteredElderlyList(p)
+activate Model
+
+loop for every elderly
```

Comment:
"not all elderlies tested" is better? Not too sure though my UML abilities are....

Reply:
This is in reference to the loop [condition]